src/crimson/os/seastore/onode_manager/staged-fltree/node_extent_accessor.h

```diff
@@ -561,7 +561,9 @@ class NodeExtentAccessorT {
         })
       );
     }).si_then([this, c] {
-      assert(!c.t.is_conflicted());
+      // FIXME: interruptive-future failed to check invalidation
```

Comment:
since this commit is marked "Fixes: https://tracker.ceph.com/issues/52259", shall we create another tracker ticker for tracking this issue ?

Reply:
Yeah, I'll add it to the tracker.

Reply:
https://tracker.ceph.com/issues/52275

Reply:
thanks, raising its priority to "high".